packages/contentful/migrations/crn/users/20231011113618-add-validation-to-links.js

```diff
@@ -0,0 +1,148 @@
+module.exports.description =
+  'Adds validation to social media urls on the users';
+
+module.exports.up = (migration) => {
+  const users = migration.editContentType('users');
+
+  users.editField('linkedIn').validations([
+    {
+      regexp: {
+        pattern:
+          '^(ftp|http|https):\\/\\/(\\w+:{0,1}\\w*@)?(\\S+)(:[0-9]+)?(\\/|\\/([\\w#!:.?+=&%@!\\-/]))?$',
+        flags: null,
+      },
+      message:
+        'Please enter a valid LinkedIn URL. (e.g. https://www.linkedin.com/in/addyourname)',
+    },
+  ]);
+
+  users.editField('twitter').validations([
+    {
+      regexp: {
+        pattern:
+          '^(ftp|http|https):\\/\\/(\\w+:{0,1}\\w*@)?(\\S+)(:[0-9]+)?(\\/|\\/([\\w#!:.?+=&%@!\\-/]))?$',
```

Comment:
Do we need to have more strict regexp patterns for these links, as it looks like a user could input any url?